### PR TITLE
Resolve "Redirect to CMS-page if Cookies are Disabled" is "No" but it still redirect issue25148

### LINKED ISSUE #25148: "Redirect to CMS-page if Cookies are Disabled" is "No" but it still redirect
### Preconditions (*)

1. Magento 2.3-develop

### Steps to reproduce (*)

1. Go to backend
2. Store-> Configuration
3. General->Web
4. "Browser Capabilities Detection" , "Redirect to CMS-page if Cookies are Disabled" is "No"

image

5. Go to frontend
6. Disable cookie
7. Signin customer

### Expected result (*)

1. No redirect to Cookie CMS-page

### Actual result (*)

1. Redirect to Cookie CMS-page

### PR DESCRIPTION
**Pull request by edenduong**
### Description (*)

1. Resolve magento/magento2#25148: "Redirect to CMS-page if Cookies are Disabled" is "No" but it still redirect

### Fixed Issues (if relevant)

1. magento/magento2#25148: "Redirect to CMS-page if Cookies are Disabled" is "No" but it still redirect

### Manual testing scenarios (*)

1. Go to backend
2. Store-> Configuration
3. General->Web
4. "Browser Capabilities Detection" , "Redirect to CMS-page if Cookies are Disabled" is "No"

image

5. Go to frontend
6. Disable cookie
7. Signin customer

#### Expected result

1. No redirect to Cookie CMS-page
We should "ALERT" to notify to the customer.

image

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by swnsma — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @swnsma, thank you for the review.
[ENGCOM-6129](https://jira.corp.magento.com/browse/ENGCOM-6129) has been created to process this Pull Request
:eight_spoked_asterisk: @swnsma, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed 

**Before:**
before

**After:**
after

**Review by VladimirZaets — DISMISSED** [review]
Hi @edenduong. Thanks for collaboration. Due to [Magento Definition of Done](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing_dod.html) all code must be covered by tests. Please cover your fix by automated tests.

**Comment by edenduong** [comment]
> Hi @edenduong. Thanks for collaboration. Due to [Magento Definition of Done](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing_dod.html) all code must be covered by tests. Please cover your fix by automated tests.

Hi @VladimirZaets, thank you for your feedback. Do you know how to disable the cookie by code? Because now to test it, I disabled it at browser so it can not run by Automation Test.

**Comment by edenduong** [comment]
Hi @VladimirZaets, I just covered my code by Unit Test. Thanks!

**Review by swnsma — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @swnsma, thank you for the review.
[ENGCOM-6129](https://jira.corp.magento.com/browse/ENGCOM-6129) has been created to process this Pull Request
:eight_spoked_asterisk: @swnsma, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

File: app/code/Magento/Cookie/i18n/en_US.csv [hunk 1]
```diff
@@ -11,3 +11,5 @@
 "Cookie Domain","Cookie Domain"
 "Use HTTP Only","Use HTTP Only"
 "Cookie Restriction Mode","Cookie Restriction Mode"
+"Cookies are disabled in your browser.","Cookies are disabled in your browser."
```
Comment (DrewML) [inline app/code/Magento/Cookie/i18n/en_US.csv:14]: We should work on this copy a bit more.

Concerns:

1. A shopper might not be familiar with what a cookie is
2. There is no mention that the store won't work without cookies
3. No direction for a shopper for how to address the issue

Would be great if we can make it clear that the store does not work without support for cookies, and maybe link to an article advising how to address that in common browsers.

**Pull request merged**